README.md

```diff
@@ -31,6 +31,10 @@ When `yes`, this variable includes and execute
 during the play, which makes it possible to manage certificates without ugly
 hacks. This is only supported in `ansible` version _at least_ 2.2 and later.
 
+`reallyenglish.x509` must be listed in `requirements.yml`, or available for
```

Comment:
this is a dangerously complicated issue for non-natives. but i will explain why I used comma here.
   
`or` has a few meanings. in _"A human being is biologically a male or a female_", I would not use a comma because the objects are equivalent alternatives. one of the other usages is to explain a preceding word. In _He is a male, or she-male_, I definitely use comma, because by using comma before `or`, I am meaning that he is a transgender.

the phrase in question meant that _"you do not necessarily need to use the method, but make sure it is available for ansible"_, rephrasing that listing in `requirements,yml` is not the point but the phrase after `or` is. `listed` is not an equivalent alternative to `available`. `listed` is a subset of `available`, or `listed` ⊂ `available`.